lib/src/utils/strings.dart

```diff
@@ -16,11 +16,32 @@ class Str {
       email: 'matthewsmith@join.com',
       phoneNumber: '+ 1 (246)869 453 00',
       bio: Str.bioText,
-      experienceTitle: 'Creative Director',
-      experiencePlace: 'at Uber',
-      experiencePeriod: 'Sep 2018 - Jan 2020',
-      experienceLocation: 'New york City',
-      experienceDescription: Str.myRoleText,
+      experience: [
```

Comment:
Great idea @talha828 thanks ✅

Reply:
when I was working on your project I was pretty impressed with your coding style. it`s is a very well define and proper code